crates/parser/src/zombienet.pest

```diff
@@ -0,0 +1,104 @@
+/// Helpers
+WHITESPACE = _{ " " | "\t" | "\r" | "\n" }
+/// matches permited node's names
+name = @{ (ASCII_ALPHANUMERIC | "-")+ }
+/// matched any integer
+int   = @{ "-" ? ~ ("0" | '1'..'9' ~ '0'..'9' * ) }
+/// matches any char
+char = @{  'a'..'z' | 'A'..'Z' | "." | "_" | "/" }
+/// matches anything between 2 double quotes
+double_quoted_string  = @{ "\"" ~ (!("\"") ~ ANY)* ~ "\""}
+/// matches anything between 2 single quotes
+single_quoted_string  = @{ "\'" ~ (!("\'") ~ ANY)* ~ "\'"}
+/// matches anything between 2 backquotes\backticks
+backquoted_quoted_string  = @{ "`" ~ (!("`") ~ ANY)* ~ "`"}
+/// matches a single colon
+colon = { ":" }
+/// matches uri
+uri = @{ "http" ~ "s"? ~ "://" ~ (ASCII_ALPHANUMERIC| "/" | "-" | "_" | ".")+ }
+/// matches any file path
+file_path = @{ "."{0,2} ~ "/" ~ (ASCII_ALPHANUMERIC | "-" | "_" | "." )+ }
+/// matches prometheus metrics labels
+metric_key_value = @{ (ASCII_ALPHANUMERIC | "_" | "-")+ ~ "=" ~ double_quoted_string }
+/// matches Jaeger span ids
+span_id = { ASCII_HEX_DIGIT{32} }
+
+/// METRICS
+block_height = { "blockheight" | "block height" | "best block" }
+finalized_height = { "finalised height" | "finalised block" }
+peers_count = { "peers count" | "peers" }
+
+metric_name = @{ (block_height | finalized_height | peers_count | (ASCII_ALPHANUMERIC | "_" )+) ~ ("{" ~ metric_key_value ~ "}")? }
+square_brackets_strings = {
+    "[" ~ double_quoted_string ~ ("," ~ double_quoted_string)* ~ "]"
+}
+match_type = { "regex" | "glob" }
+
+/// OPERATORS
+op_lte       = { "<=" }
+op_gte       = { ">=" | "is at least" | "at least" }
+op_lt        = { "<" | "is lower than" }
+op_gt        = { ">" | "is greater than" | "greater than" }
+op_eq        = { "==" | "=" | "is equals" | "is equals to" | "is" }
```

Comment:
Im not sure that `is equals` and `is equals to` is correct... 
shouldn't it be:
```suggestion
op_eq        = { "==" | "=" | "equals" | "equals to" | "is" }
```
?

Reply:
The actual `DSL` have support for `is equals` and `is equals to`. We can also support `equals` and `equals to`.
Thanks!

Reply:
It should be `is equal` and `equals`?

Reply:
I'd say `is equal` and `equals to`

Reply:
I checked in `polkadot` and `cumulus` repos and we never use the `equals` operator (in any of their version :))

we can go with any of:
```
charlie: reports block height is equal 2
charlie: reports block height equals 2
```

```
charlie: reports block height is equal 2
charlie: reports block height equal to 2
```

Reply:
`equal to` is grammatically wrong I think. Shouldn't it be `equals to`?

Reply:
yes, I think that correct. Thanks!